app_dart/README.md

```diff
@@ -4,19 +4,24 @@ This folder contains a Dart based backend for Cocoon.
 
 ## Building and running
 
-#### Prerequisites
+### Prerequisites
 
-You'll need to install the Google Cloud Developer Tools Command Line Interface
-(`gcloud`) in order to do several of the tasks below. Visit
-https://cloud.google.com/sdk/docs/quickstarts for instructions on how to install
-these tools for your platform.
-
-Upon downloading and installing the Google Cloud SDK, you should initialize it
+* Install the Google Cloud Developer Tools Command Line Interface
+([`gcloud`](https://cloud.google.com/sdk/docs/quickstarts)). Then initialize it
 and authenticate yourself by running:
 
 ```sh
+gcloud auth login
 gcloud init
 ```
+* Install Flutter
```

Comment:
Should we just redirect to https://flutter.dev/docs/get-started/install ? We skipped this at first when web was experimental and required the config flag below.

Reply:
Added the link.
I received complains about missing `flutter upgrade` and `flutter pub get` when people refer to the internal doc to develop locally. So prefer to keep them here.